openapi.yaml

```diff
@@ -692,7 +693,7 @@ paths:
                             schema:
                                 allOf:
                                     - $ref: "#/components/schemas/VisibilityInfo"
-                                    - x-speakeasy-wrapped-attribute: "visibility_info"
```

Comment:
if we stop wrapping it, this will return values at the same level as the parameters specified in the tf block. They have the same type so this will let the user actually get refreshed state and compare their changes against remote